Workbooks/WorkbooksMetadata.json

```diff
@@ -1398,7 +1398,7 @@
       "dataConnectorsDependencies": [],
       "previewImagesFileNames": [ "AzureSentinelCostWhite.png", "AzureSentinelCostBlack.png"],
       "version": "1.2",
```

Comment:
Please update the version as well. Thanks

Reply:
Hi @v-sumdh, 

Got it. Updated the version to 1.3.

thanks